src/ui/public/test_harness/test_harness.js

```diff
@@ -6,6 +6,7 @@ import chrome from 'ui/chrome';
 import Nonsense from 'Nonsense';
```

Comment:
Since the courier lives outside of angular, it's state is not reset in between tests. With the new valid check in #8723 calls to `notify.error()` were breaking the notifier tests, so I figured it would be a good idea to fail a test if it left notifications in the notifier (tests need to clean up after themselves).